src/common/mem.c

```diff
@@ -2213,7 +2239,7 @@ umem_cache_free(struct umem_store *store)
 static inline unsigned int
 cache_mode(struct umem_cache *cache)
 {
-	return (cache->ca_md_pages == cache->ca_mem_pages) ? 1 : 2;
+	return cache->ca_mode;
```

Comment:
I don't think this change is necessary, no matter if DAV v1 or v2 is used, the cache mode is always decided by "md_pages == mem_pages", right?

Reply:
As discussed offline, the mode is decided by the allocator type.